WordPress/src/main/res/layout/publicize_list_fragment.xml

```diff
@@ -51,7 +51,7 @@
                 android:textColor="@color/grey_dark"
                 android:textSize="@dimen/text_sz_medium" />
 
-            <org.wordpress.android.widgets.WPButton
+            <Button
```

Comment:
Sorry, didn't mean size, I meant background. Yea, I made the change, just wanted to make sure it was conscious that it wasn't touched 👍